snippets/Google.Datastore.V1Beta3.Snippets/DatastoreDbSnippets.cs

```diff
@@ -155,16 +154,16 @@ public void GqlQuery()
             // DatastoreQueryResults implements IEnumerable<Entity>, but you can
```

Comment:
Ack. (I'll search for AsBatches, AsResponses, `IEnumerable<Entity>` and `AsEntityResults` to try to find all stray comments.) Will probably only *do* after receiving other comments though, rather than piecemeal.